types/block.yaml

```diff
@@ -1,52 +1,69 @@
 schemas:
+  BeaconBlockCommon:
+    # An abstract object to collect the common fields between the BeaconBlockHeader and the BeaconBlock objects
```

Comment:
My bad on bumping to v0.11.0 links without fixing params.

Is the plan here to get v0.10.x cleaned up and release. Then update to v0.11?

Reply:
I though 0.10.x was going to be targeted version, I was wrong :smile: 
So yeah, I think it makes sense to update incrementally